recipes/gpustat/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "gpustat" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2f43421dbcd9ebd8caf179aeb5f78ac123786033fb9a4310ce0f8e18b25eb03e
+
+build:
+  skip: True  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - gpustat = gpustat:main
+
+requirements:
+  build:
+    - setuptools
+  host:
+    - python
+    - pip
+    - pytest-runner
+    - nvidia-ml
```

Comment:
Are these all needed? The host section only needs the dependencies required for running the setup.py file.